chia/wallet/vc_wallet/vc_wallet.py

```diff
@@ -424,6 +431,185 @@ async def revoke_vc(
         else:
             return [tx]  # pragma: no cover
 
+    async def add_vc_authorization(
+        self, offer: Offer, solver: Solver, reuse_puzhash: Optional[bool] = None
+    ) -> Tuple[Offer, Solver]:
```

Comment:
Since this is a pretty complex function a docstring outlining what it's doing would be good

Reply:
Added